ansible/group_vars/all.yml

```diff
@@ -668,7 +669,6 @@ enable_neutron_port_forwarding: "no"
 enable_nova_serialconsole_proxy: "no"
 enable_nova_ssh: "yes"
 enable_octavia: "no"
-enable_octavia_driver_agent: "{{ enable_octavia | bool and neutron_plugin_agent == 'ovn' }}"
```

Comment:
I think this came from a backport?

Reply:
Backported from a patch in progress in #151, which also caused a merge conflict later. I propose a revert, synchronisation with the upstream and backporting https://review.opendev.org/c/openstack/kolla-ansible/+/783699 again